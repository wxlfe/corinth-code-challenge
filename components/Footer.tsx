import { Container, Dropdown, Link, Spacer, Text } from '@nextui-org/react';

const Footer = () => {
  return (
    <footer>
      <hr />
      <Container
        fluid
        gap={0}
        justify='space-around'
        css={{ d: 'flex', padding: '1rem' }}
      >
        <Link href='https://wxlfe.dev' color='primary'>
          <Text h3 color='primary'>
            Created by Nate Wolfe
          </Text>
        </Link>
      </Container>
    </footer>
  );
};

export default Footer;
