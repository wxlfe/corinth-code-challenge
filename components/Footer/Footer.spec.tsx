import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import Footer from './Footer';

describe('Footer Component', () => {
  test('should create', () => {
    expect(Footer).toBeTruthy();
  });

  test('should render', () => {
    const { asFragment } = render(<Footer></Footer>);
    expect(asFragment()).toMatchSnapshot();
  });

  test('should contain portfolio link', () => {
    const { asFragment } = render(<Footer></Footer>);
    expect(asFragment()).toHaveTextContent('Created by Nate Wolfe');
  });
});
