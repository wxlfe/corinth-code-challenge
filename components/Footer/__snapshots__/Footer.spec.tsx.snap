// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should render 1`] = `
<DocumentFragment>
  <footer>
    <hr />
    <div
      class="nextui-c-fdHeMm nextui-c-fdHeMm-cFpcyF-fluid-true nextui-c-fdHeMm-dwxLNB-responsive-true nextui-c-fdHeMm-iiuOaOp-css"
    >
      <a
        class="nextui-c-dIWzTP nextui-c-PJLV nextui-c-dIWzTP-hQWhoE-color-primary nextui-c-dIWzTP-isWIbj-animated-true nextui-c-PJLV-gulvcB-isFocusVisible-false nextui-c-dIWzTP-ivnGVk-css nextui-link"
        href="https://wxlfe.dev"
      >
        <h3
          class="nextui-c-PJLV nextui-c-PJLV-ihoHSOW-css"
        >
          Created by Nate Wolfe
        </h3>
      </a>
    </div>
  </footer>
</DocumentFragment>
`;
